pyprof/nvtx/nvmarker.py

```diff
@@ -267,22 +288,76 @@ def modMarker(mod, fn_name, args):
 
 
 def add_wrapper(mod, fn_name):
-    assert isfunc(mod, fn_name)
+    # assert isfunc(mod, fn_name)
```

Comment:
why disabled?

Reply:
There is an assumption in this function that only funcs will call this.  The value of this is that it asserts this truth and protects against this function being called incorrectly. Is there a lot of overhead inside of the function?

Reply:
`isfunc` contains 7 function calls and a linear search through an array to check inclusion in a list. It is none trivial.  
There are two instances of `add_wrapper` function calls and both are preceded with an `if isfunc` check. The assertion is entirely redundant.

Reply:
Under normal circumstances I think we want to keep protective code like this.  However, given that we are trying to streamline this file I'm ok to remove it.